addons/l10n_cl_pos/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="res_config_settings_view_form_inherit" model="ir.ui.view">
+            <field name="name">res.config.settings.pos.cl</field>
+            <field name="model">res.config.settings</field>
+            <field name="inherit_id" ref="point_of_sale.res_config_settings_view_form"/>
+            <field name="priority" eval="8"/>
+            <field name="arch" type="xml">
+                <field name="pos_journal_id" position="after">
+                    <field name="country_code" invisible="1"/>
+                </field>
+                <field name="pos_journal_id" position="attributes">
```

Comment:
Shouldn't this change should be in 'point_of_sale'? As it seems a good idea, and once the localisation CL is installed it is applied for all pos_config of any companies.